### DISCUSSION TITLE
Add Comment among XML code

### DISCUSSION THREAD

**Discussion by phong-mblab** [oozcitak/xmlbuilder2#137] [Q&A] [answered]
Hi, besides XML tags, can I add comments between them to describe the tag's functionality?

**Comment by oozcitak** [comment, marked as answer]
Yes. See: https://oozcitak.github.io/xmlbuilder2/node-creation-functions.html#com and https://oozcitak.github.io/xmlbuilder2/object-conversion.html#comment
